Makefile

```diff
@@ -133,3 +133,6 @@ endef
 
 help: ## Display this help.
 	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_0-9-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
+
+test-validators:
```

Comment:
We can drop this target I think. Let's modify `make test` as needed, since this is what `pr_check.sh` will be running. Right now it is `$ go test -count=1 $(PKGS)`

Reply:
Ohh gotcha! Purging this, thanks :)